src/components/custom-field-input-number/__snapshots__/custom-field-input-number.test.jsx.snap

```diff
@@ -45,6 +39,6 @@ Object {
   "id": "test-input",
   "name": "field-id",
   "validity": ValidityState {},
-  "value": "",
+  "value": "undefined",
```

Comment:
Because we're changing the API of an existing component, this is a "breaking change". We will _need_ a CI of bigmaven with these changes. 

Please update the PR upkeep list in the PR description.